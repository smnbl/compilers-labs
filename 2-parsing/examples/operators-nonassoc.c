int operators_nonassoc()
{
    x == y == z;
}
