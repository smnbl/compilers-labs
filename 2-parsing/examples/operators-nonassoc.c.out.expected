parser: error: 3:12: non-associative operators may not be used multiple times in a row
